src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/SendRequestNewTrainingProvider.cshtml

```diff
@@ -0,0 +1,54 @@
+﻿@using SFA.DAS.CommitmentsV2.Shared.Extensions
+@model SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice.SendNewTrainingProviderViewModel
+
+@{
+    ViewBag.GaData.Vpv = "/change-provider/send-request-new-training-provider";
+    ViewBag.GaData.Org = Model.EmployerName;
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <partial name="_ValidationSummary" />
+        <form method="post">
+            @Html.HiddenFor(x => x.NewProviderName)
+            @Html.HiddenFor(x => x.ProviderId)
+            @Html.HiddenFor(x => x.AccountHashedId)
+            @Html.HiddenFor(x => x.ApprenticeshipHashedId)
+            <h1 class="govuk-heading-xl">Are you sure you want to send this request to @Model.NewProviderName?</h1>
+
+            @if (Model.ApprenticeshipStatus != SFA.DAS.CommitmentsV2.Types.ApprenticeshipStatus.Stopped)
+            {
+                <p class="govuk-body das-!-colour-inherit">
+                    Once you send this request, @Model.ApprenticeName's apprenticeship record and payments to @Model.OldProviderName will be stopped
```

Comment:
if the name ends in 's' such as "Jones", should be `'` not `'s`

Reply:
Nice catch, never crossed my mind. Made changes for this.